nss-day-cohort-54/worst-website-ever-group-4
# added annoying CSS to footer

Added obnoxious css file for stylesheets
Added animation css to footer content

**Steps to test
1. Check to see if stylesheet file was created and uploaded properly
2. Confirm that all three links in footer are animated
